server/rpc/gateway.go

```diff
@@ -0,0 +1,116 @@
+package rpc
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
+	"github.com/medibloc/panacea-oracle/config"
+	"github.com/medibloc/panacea-oracle/server/service/datadeal"
+	"github.com/medibloc/panacea-oracle/server/service/key"
+	"github.com/medibloc/panacea-oracle/server/service/status"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/backoff"
+)
+
+type gatewayServer struct {
```

Comment:
Intellij say `Exported function with the unexported return type`.
```suggestion
type GatewayServer struct {
```

Reply:
Since this struct doesn't need to be known from the outside, it can be hidden by starting with a lowercase letter.

Reply:
Well I'm not getting the error. that's weird...

Reply:
Because the grpc struct also starts with a lowercase letter.

https://github.com/medibloc/panacea-oracle/pull/79/files#diff-a747d2668c12198a63a914995f378fbcf063c14f83175279b2328a62b8947a36R20

Reply:
Sorry for confusing you.

`NewGrpcServer()` returns the struct, that is the struct is used outside the package. That's why intellij warn me I think.

https://github.com/medibloc/panacea-oracle/pull/79/files#diff-a747d2668c12198a63a914995f378fbcf063c14f83175279b2328a62b8947a36R25

Reply:
I understand about your comment.
I fixed it.(https://github.com/medibloc/panacea-oracle/pull/79/commits/4df1a1c0387f42ee463bfda4a39a110ae03ef53c)
Thank you